lti_consumer/lti_1p3/key_handlers.py

```diff
@@ -68,8 +68,15 @@ def _get_keyset(self, kid=None):
         keyset = []
 
         if self.keyset_url:
-            # TODO: Improve support for keyset handling, handle errors.
-            keyset.extend(load_jwks_from_url(self.keyset_url))
+            try:
+                keys = load_jwks_from_url(self.keyset_url)
+            except Exception as err:
```

Comment:
@felipetrz This will fail quality checks (exception too generic). Can you add and exception here?

Reply:
@giovannicimolin This does not fail pylint checks because I'm raising another chained exception from the general Exception, in this case.
Adding a disable here results in a `useless-suppression` warning.